client/features/sharedComponents/SearchBarActive.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import SearchBar from '../sharedComponents/SearchBar';
 import activeMessage from '../sharedComponents/helpers/activeMessage';
-import TMDB_image from '../TMDB/TMDB_image';
+import TMDBimage from '../TMDB/TMDB_image';
```

Comment:
Does that work? You might have to rename the class if you want it without the underline.

Reply:
yeah! I renamed the class in the file. Wasn't sure if I should rename the file name too. The only reason I did so is because of pomander.